templates/zilencer/billing.html

```diff
@@ -30,11 +30,15 @@ <h1>{{ _("Billing") }}</h1>
                     <div class="tab-pane active" id="overview">
                         <p>Your current plan is <strong>{{ plan_name }}</strong></p>
                         <p>You are paying for <strong>{{ seat_count }} users</strong>.</p>
-                        <p>Your plan will renew on <strong>{{ renewal_date }}</strong> for <strong>${{ renewal_amount }}</strong>.</p>
-                        {% if prorated_charges %}
-                        <p>You have <strong>${{ prorated_charges }}</strong> in prorated charges that will be added to your next bill.</p>
-                        {% elif prorated_credits %}
-                        <p>You have <strong>${{ prorated_credits }}</strong> in prorated credits that will be automatically applied to your next bill.</p>
+                        {% if cancel_at_period_end %}
+                            <p>Your subscription for Zulip Premium is ending on <strong>{{ current_period_end }}</strong>. You would be downgraded to Zulip Free when the subscription ends.</p>
+                        {% else %}
+                            <p>Your plan will renew on <strong>{{ current_period_end }}</strong> for <strong>${{ renewal_amount }}</strong>.</p>
+                            {% if prorated_charges %}
```

Comment:
The prorated charges/credits apply even if the subscription is being canceled. There are some policy things that need to be decided here though (e.g. what happens to unused credits when you cancel), so I'm tempted to hold on this for now.